pages/categories/downstream.md

```diff
@@ -12,7 +12,7 @@ Downstream emissions are related to GHG Protocol Scope 3.
 
 {% include linkedHeading.html heading="End-User Devices" level=2 %}
 
-This considers the emissions generated from the electricity consumption of devices such as desktops, laptops, tablets, and mobile phones that utilise the products or services provided by the organisation. 
+This considers the emissions generated from the electricity consumption of devices such as desktops, laptops, tablets, and mobile phones that utilise the products or services provided by an organisation. It is important to account for the differences in energy efficiency across various devices accessing these products and services. Typically, a smartphone uses less energy than a laptop, and a laptop uses less energy than a desktop. Understanding the [energy use of devices](/information/lifecycle/usage) can then be used to estimate carbon emissions by considering the source of the energy ([Carbon Intensity](/glossary#carbon-intensity)) used to power or charge that device. Finally, end-user devices should have their embodied carbon estimated. This should take into account the product’s life span to consider the entire [Product Life Cycle](/glossary#product-life-cycle). [Read more about hardware lifecycle emissions](/information/lifecycle).
```

Comment:
Apologies as I can't quite remember the conclusion we came to on this - but is the embodied carbon of end-user devices always in scope? For example, if the end-user is accessing an application produced by the organisation - should downstream refer only to the associated usage emissions from accessing this, assuming the user already owns that device and hasn't bought it specifically for the purpose of using the organisation's product (i.e. the embodied carbon sits with them).

Reply:
or due to double counting being fine for scope 3, does downstream consider all emissions associated with end-user devices?

Reply:
Yeah, I think that's about right. There's some space to mention backward compatibility to ensure people aren't forced to upgrade devices to use our product but I don't think we're wanting to include a slice of amortised embodied carbon in our estimation?

Reply:
You're right... I should re-word this better. 
Basically, no - an organisation wouldn't account for the end user device's embodied carbon - but they should _consider_ the consequential effects of the changes the org makes. ie, does the software require the users to upgrade their hardware in order to use it.